sbin/root-mount

```diff
@@ -53,27 +53,19 @@ LVM_RESERVED_KB=$(expr $LVM_RESERVED_MB \* 1024)
 
 pwr_key_wait()
 {
-	while true; do
-		key=$(/sbin/evkey -d -t 3000 /dev/input/event6)
-
-		if [ "x${key}" == "x116" ]; then
-			key=null
-			key=$(/sbin/evkey -u -t 2000 /dev/input/event6)
-
-			if ! [ "x${key}" == "x116" ]; then
-				# Turn on RED led
-				echo 255 > /sys/class/leds/led:rgb_red/brightness
-				# Kill UI process
-				kill $1
-				# Let user see the red LED for couple of seconds.
-				sleep 2
-				# clear any pending RTC wake up so we don't wake up
-				rtc-clear
-
-				poweroff -f
-			fi
-		fi
-	done
+	# Waiting for Power key pressed for 3 seconds and released.
+	if /usr/bin/yamui-powerkey -u; then
+		# Turn on RED led
+		echo 255 > /sys/class/leds/led:rgb_red/brightness
```

Comment:
This is device specific. We need to have configuration each device which path to echo and what I guess. Some devices might not even have led.